src/routes/MainRoute/MainRouteStyled.js

```diff
@@ -0,0 +1,141 @@
+import styled from 'styled-components';
+import SmallLogo from '@/images/small_logo.svg';
+import Miniature from '@/images/main_page_miniature.svg'
+import { Typography, Box } from '@mui/material';
+
+export const MainPageLayout = styled(Box)`
+box-sizing: border-box;
+padding-top: 80px;
+font-family: Roboto, sans-serif;
+`
+
```

Comment:
It is beeter to use ";" in the end of each styled component. It just make it easier to read and support the file itself.

Reply:
Noted. fixed and will look why prettier stopped working on these files.